### DISCUSSION TITLE
Applying Tooltip to a Card

### DISCUSSION THREAD

**Discussion by tomsoja978** [primefaces/community#1005] [PrimeReact] [answered]
I can't get Tooltips working with Cards. Are they not compatible?

https://primereact.org/tooltip/
https://primereact.org/card/

If there is a way (or workaround), I'd appreciate a beginner-friendly, detailed, step-by-step guide as i am very new to this world. Thank you so much.

**Comment by melloware** [comment, marked as answer]
Here you go: https://stackblitz.com/edit/eegdrz?file=src%2FApp.jsx,src%2Fmain.jsx

**Reply by tomsoja978** [reply]
Thank you so much
